internal/pkg/template/templates/environment/cf.yml

```diff
@@ -557,6 +557,62 @@ Resources:
 {{include "lambdas" . | indent 2}}
 {{include "custom-resources" . | indent 2}}
 {{- end}}
+{{- if .VPCConfig.Flowlogs }}
+  Flowlog:
+    Metadata:
+      'aws:copilot:description': 'A Flowlog for the VPC to capture information about the IP traffic'
```

Comment:
nit
```suggestion
      'aws:copilot:description': 'a flow log for the VPC to capture information about the IP traffic'
```

Reply:
Normally the first letter is uppercase, isn't it?

Reply:
typo! fixed!